docs/future-work/oss-communication-improvements/build-getting-started-tutorial.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Build Getting Started Tutorial
+rfc: not-required
+estimated-scope: medium
+improved-metric: community-engagement
+skills: video-production, writing, tutorial-creation, documentation
```

Comment:
I wouldn't say tutorial-creation is a skill. Do we need that? Also documentation and writing seem redundant, could we just pick one of the two and use consistently?